Lion.java

```diff
@@ -1,13 +1,35 @@
-//Animal class. Currently only used to test if the parent class methods are functional
+//Animal, one of the options that can be chosen by users
 
-public class Lion extends Animal {
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
 
-  public Lion(String name, int strengthScore, int weightScore, int agilityScore,
-              int climbingScore, int warmthScore, int hungrinessScore,
-              int stealthScore, int cunningScore, int clawSharpnessScore) {
-    super(name, strengthScore, weightScore, agilityScore, climbingScore,
-          warmthScore, hungrinessScore, stealthScore, cunningScore);
-          
-    attributes.put(Attributes.CLAW_SHARPNESS, new AttributeValue("Claw Sharpness", clawSharpnessScore));
+public class Lion extends Animal implements IDescribable {
+  
+  static ArrayList<Attributes> requiredLionAttributes = new ArrayList<Attributes>(Arrays.asList(Attributes.CLAW_SHARPNESS));
+
+  public Lion(String name) {
+    super(name);    
+  }
+
+  //Used to populate the HashMap
+  public void setAttributes(ArrayList<AttributeValue> values) {
```

Comment:
why do you have to override this function... the base function should do the job.